### PR TITLE
Loads the game from the server

### PR DESCRIPTION
**Pull request by AnthonyDario**
When you press the "Test JSON" button, If the server is running the game will load the data from the server.

### CODE REVIEW

**Comment by CatalystOfNostalgia** [comment]
@AnthonyDario You have merge conflicts. Always pull master before submitting a pull request =).

**Comment by AnthonyDario** [comment]
@CatalystOfNostalgia Fixed!

File: client/poof_game/assets/Scripts/src/GameObjects/Tile.cs [hunk 1]
```diff
@@ -25,14 +28,29 @@ public class Tile : MonoBehaviour {
...
-    {
-        isVacant = true;
-    }
+	public Building PlaceBuilding(Building newbuilding) 
+	{
+		if (isVacant) {
+
+			building = Instantiate (newbuilding, 
+			            			new Vector3(this.transform.position.x, 
+			            			 			this.transform.position.y - .65f,
```
Comment (CatalystOfNostalgia) [inline client/poof_game/assets/Scripts/src/GameObjects/Tile.cs]: plz fix indent. Also refactor the constant into a config

File: client/poof_game/assets/Scripts/src/Managers/GameStart.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ public class GameStart : MonoBehaviour {
 	void Awake () {
         Instantiate(grid, new Vector3(0, 0, 15), Quaternion.identity);
         Instantiate(gManager, new Vector3(0, 0, 0), Quaternion.identity);
-		Instantiate (bManager, new Vector3 (0, 1, 0), Quaternion.identity);
+		Instantiate(bManager, new Vector3 (0, 1, 0), Quaternion.identity);
```
Comment (CatalystOfNostalgia) [inline client/poof_game/assets/Scripts/src/Managers/GameStart.cs]: Fix indent

**Comment by AnthonyDario** [comment]
@CatalystOfNostalgia good now?

**Comment by CatalystOfNostalgia** [comment]
Looks good. Merged.

**Pull request merged**